.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @arithx @lucab
```

Comment:
@arithx FYI this may be noisy (default assign for review), but I don't expect many PR after the initial development. Let me know if not ok.

Reply:
Yeah, I noticed it, but it works for me.